ozaria/site/components/teacher-dashboard/BaseSingleClass/table/TableModuleHeader.vue

```diff
@@ -9,6 +9,37 @@ import LockOrSkip from './LockOrSkip'
 
 import { mapGetters, mapMutations } from 'vuex'
 
+// The levels in the groups defined here are can not be selected individually.
+// They are either all selected or all not selected.
+const selectableGroups = [
+  [
+    // Chapter 1
+    '5efc08940bda4700242d6e3c', // Intro: Trapping Darkness
+    '5efc09910bda4700242d6e47', // Intro: Builder Things
+    '5efc10a40bda4700242d6e9b', // Intro: Finishing Touches
+    '5eddd6a76f7d690028cf4c50', // Capstone Level: Gauntlet
+    '5f0cd339e494e50029521c5e' // Cutscene: Trapping the Dark
+  ],
+  [
```

Comment:
i'm not so familiar with ozaria levels, is there no capstone projects in chap 2 and 3?

Reply:
@smallst I'm not sure either. Task description says:
> Chapter 1 and 4 capstones have issues.